src/pageContent/Profile/index.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+
+import Button from "src/components/Button";
+import Input from "src/components/Input";
+import Text from "src/components/Text";
+import Image from "next/image";
+
+import * as S from "./styles";
+
+const ContactUsContent: React.FC = () => {
+  const [name, setName] = useState<string | null>(null);
+  const [subject, setSubject] = useState<string | null>(null);
+  const [content, setContent] = useState<string | null>(null);
+
+  const handleSubmit = (event: React.MouseEvent) => {
+    event.preventDefault();
```

Comment:
What would happen if I dont have `preventDefault`

Reply:
Supposedly it will do the default behaviour upon clicking? But honestly idk what default behaviour is being prevented here.

Reply:
the ans is : the default behaviour of button is doing nothing